packages/netlify-cms-locales/src/ja/index.js

```diff
@@ -111,8 +111,8 @@ const ja = {
     },
     editorWidgets: {
       markdown: {
-        richText: 'リッチテキスト',
-        markdown: 'マークダウン',
+        richText: 'Rich Text',
```

Comment:
Any reason to change these?
Maybe we can increase the width of that button.
Regarding other choices (headings, image, block) shouldn't those be translated too?

Reply:
@erezrokah 
thank you for your kind suggestion!

I increased the buttons width to fit the contents and translated headings. 🙏 
image and block are a little difficult because we cannot handle adding custom widget like youtube...

Reply:
I think this is already a great improvement, thanks!